1907-01-21.xml

```diff
@@ -1494,13 +1497,973 @@
                         <p>G.R. MACAULEY. General Manager</p>
                     </div>
                     </div>
-                </div>
-
                 <div type="page" n="2"> </div>
```

Comment:
move the `</div>` tag, which closes page 2 as soon as you've opened it, to the end of the page, i.e. just before you start page 3.